gnocchi/storage/__init__.py

```diff
@@ -31,6 +31,10 @@
     cfg.StrOpt('driver',
                default='file',
                help='Storage driver to use'),
+    cfg.BoolOpt('enable_telemetry', default=True,
```

Comment:
I don't understand the reasoning behind this option. It sounds kinda useless to me but maybe there's something I miss.

And the option name is completely "misleading" to me.

Reply:
if i don't add this, and you actually hit the warning, the warning will get thrown forever as there's no way you can disable the warning... i could add a cache to capture stuff we've already logged but a metric can be handled by multiple workers so it'll still log multiple times.... and if we ever expand telemetry, it'll be more stuff we need to cache.

i'm open to another approach or rename.

Reply:
I see, thanks for the explanation.

I'd say add a setting to configure this 320000 size so it can be set to `None` to be disabled. Something like `warn_backwindow_size`.

And I hope we'll be able to kill that option later. :)